<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="GL3888-X0" failures="2" tests="4" name="Test" time="8.914" errors="0" timestamp="2024-09-02T01:40:48 IST">
  <testcase classname="api.test.UserTests" name="testPostUser" time="2.802"/>
  <testcase classname="api.test.UserTests" name="testGetUserByUserName" time="1.514">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;404&amp;gt; but was:&amp;lt;200&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<404> but was:<200>
at api.test.UserTests.testGetUserByUserName(UserTests.java:50)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testGetUserByUserName -->
  <testcase classname="api.test.UserTests" name="testUpdateUser" time="2.745"/>
  <testcase classname="api.test.UserTests" name="testDeleteUserByUserName" time="1.283">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;404&amp;gt; but was:&amp;lt;200&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<404> but was:<200>
at api.test.UserTests.testDeleteUserByUserName(UserTests.java:76)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserByUserName -->
</testsuite> <!-- Test -->
